uint8 gOBJ         # Object detection state (0-3)
string gOBJ_desc   # Description

uint8 gSTA         # Gripper status (0-3)
string gSTA_desc   # Description

uint8 gGTO         # Action status
string gGTO_desc   # Description

uint8 gACT         # Activation state
string gACT_desc   # Description

uint8 kFLT         # Controller fault code
string kFLT_desc   # Description

uint8 gFLT         # Gripper fault code
string gFLT_desc   # Description

uint8 gPR          # Requested position
string gPR_desc    # Description

uint8 gPO          # Actual position
string gPO_desc    # Description

uint16 gCU         # Motor current (in mA)
string gCU_desc    # Description
